btcec/pubkey_test.go

```diff
@@ -247,3 +247,39 @@ func TestPubKeys(t *testing.T) {
 		}
 	}
 }
+
+func TestPublicKeyIsEqual(t *testing.T) {
+	pubKey1, err := btcec.ParsePubKey(
+		[]byte{0x03, 0x26, 0x89, 0xc7, 0xc2, 0xda, 0xb1, 0x33,
```

Comment:
Actually, if you'd prefer, I can just do this in a separate PR because there are other cases in the tests that should be using `decodeHex` as well.